libs/remix-ui/solidity-compiler/src/lib/compiler-container.tsx

```diff
@@ -704,6 +720,7 @@ export const CompilerContainer = (props: CompilerContainerProps) => {
             <label className="remixui_compilerLabel form-check-label" htmlFor="versionSelector">
               Compiler
               <button className="far fa-plus btn-light border-0 p-0 mx-2 btn-sm" onClick={promptCompiler} title="Add a custom compiler with URL"></button>
+              <button className="fa fa-file-text-o btn-light border-0 p-0 mx-2 btn-sm" onClick={showCompilerLicense} title="See compiler license"></button>
```

Comment:
to make those look like icons on FE we would need to:
- put span instead of button (you would need to fix the action)
- remove btn-sm
- rm btn-light
- add ml-3 to the first span to move it away of the label

and please use mr-2 instead of mx-2 on the second icon because now we have 2+2 between them

Reply:
![Screenshot from 2022-08-15 10-43-59](https://user-images.githubusercontent.com/9685356/184604707-5183f84c-8719-4e9d-98d4-5b8420e1245f.png)
![Screenshot from 2022-08-15 10-42-36](https://user-images.githubusercontent.com/9685356/184604710-fd632fd9-f9dd-41d3-93c3-413810894ad7.png)
![Screenshot from 2022-08-15 10-42-13](https://user-images.githubusercontent.com/9685356/184604714-402f8367-0986-4bce-acb2-8770a31f13f6.png)

Reply:
also I have "could not retreive license" in modal

Reply:
updated UI